src/commands/isWorkspaceInAzure.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StaticEnvironment } from "../tree/EnvironmentTreeItem";
+import { StaticWebApp } from "../tree/StaticWebAppTreeItem";
+import { tryGetBranch, tryGetRemote } from "../utils/gitHubUtils";
+
+export async function isWorkspaceInAzure(swa: StaticWebApp, env: StaticEnvironment): Promise<boolean> {
```

Comment:
I meant more like this in terms of sharing code: 
https://github.com/microsoft/vscode-azurefunctions/blob/efe121205711eafe30adfcd88e494a85745e15c0/src/tree/remoteProject/RemoteFunctionsTreeItem.ts#L24

Feels a bit weird to have this random function in "src/commands". If you don't want to do the above, at least move it to the "tree" folder, because that's where it's used.

Reply:
Ah, I see what you mean.  Yeah, I can do something like that instead.